CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 *                        @utilitywarehouse/pubsub
 /*/customer-billing/     @utilitywarehouse/org-billing-experience
 /*/finance/              @utilitywarehouse/finance-platform
+/*/customer-proposition/ @utilitywarehouse/org-multi-service-innovation
```

Comment:
We'll want utilitywarehouse/pubsub to be codeowners for the msk config so can you please remove this bit?